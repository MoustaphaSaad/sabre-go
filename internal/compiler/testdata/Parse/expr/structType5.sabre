struct {
    x, y int
    u float32