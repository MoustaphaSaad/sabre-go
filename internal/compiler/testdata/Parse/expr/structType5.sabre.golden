>> 	    u float32
>> 	             ^
Error[internal/compiler/testdata/Parse/expr/structType5.sabre:3:14]: expected '}' but found 'INVALID'
